src/utils/ucx/ucx_utils.cpp

```diff
@@ -117,41 +119,24 @@ nixlUcxEp::closeImpl(ucp_worker_h worker, ucp_ep_close_flags_t flags)
             return ucx_status_to_nixl(UCS_PTR_STATUS(request));
         }
 
-        if (worker == nullptr) {
-            ucp_request_free(request);
-            eph = nullptr;
-            return NIXL_SUCCESS;
-        }
-        break;
+        ucp_request_free(request);
+        eph = nullptr;
+        return NIXL_SUCCESS;
```

Comment:
do we need to wait for completion in certain cases? E.g. when worker is destroyed?